### ISSUE TITLE
Does group work in recipes?

### ISSUE THREAD

**Issue by diegozea** [JuliaPlots/Plots.jl#1167]
Hi! I have this recipe: https://github.com/diegozea/MIToS.jl/blob/master/src/PDB/Plots.jl
`group --> chains` inside the recipe. But it is ok when `group` is used with the same values in the `plot` call.
Best regards,

Code to reproduce it on Julia 0.6:
```julia
Pkg.add("MIToS")
Pkg.checkout("MIToS")
using MIToS.PDB, Plots; plotly()
pdbfile = downloadpdb("3JC8")
res = read(pdbfile, PDBML);
plot(res) # group inside recipe doesn't work
plot(res, group=[r.id.chain for r in res if r.id.group == "ATOM"])
```
![image](https://user-images.githubusercontent.com/2822757/31450303-64ef0d62-aea9-11e7-9913-3998112fb04e.png)

**Comment by mkborregaard** [comment]
No, you cannot use group inside a recipe. Instead manually loop over your group levels and create a new series with the `@series` macro.

**Comment by diegozea** [comment]
Thanks! I will try that :)
